serialization/src/test/scala/sbt/serialization/PicklerTypeSpec.scala

```diff
@@ -34,33 +30,33 @@ class PicklerTypeTest {
 
   @Test
   def testPickleApple: Unit = {
-    assertEquals("Apple(1)", appleExample, Apple(1).pickle.value)
+    assertEquals("Apple(1)", appleExample, SerializedValue(Apple(1)).toJsonString)
```

Comment:
Should `sbt.serialization` provide some convenience function like

```scala
sbt.serialization.toJsonString(Apple(1))
sbt.serialization.fromJsonString[Apple](appleExample)
```

It would be nice to have some symmetry between

```scala
SerializedValue(Apple(1)).toJsonString
SerializedValue.fromJsonString(appleExample).parse[Apple]
```

if we are getting rid of ops.